atomic_reactor/plugins/post_export_operator_manifests.py

```diff
@@ -81,7 +81,9 @@ def run(self):
         if not self.should_run():
             return
 
-        manifests_archive_dir = tempfile.mkdtemp()
+        os.makedirs(os.path.join(self.workflow.source.workdir, 'temp'), exist_ok=True)
+        manifests_archive_dir = tempfile.mkdtemp(dir=os.path.join(self.workflow.source.workdir,
+                                                                  'temp'))
```

Comment:
Not needed for now, you could leave an OSBS2 TBD comment that this should go in the context dir

Reply:
reverted and added comment